scripts/pre-commit.bash

```diff
@@ -0,0 +1,22 @@
+# Flutter formatter
+printf "\e[33;1m%s\e[0m\n" '=== Running Flutter Formatter ==='
+
+hasNewFilesFormatted=$(git diff --name-only)
+flutter format $hasNewFilesFormatted
+if [ -n "$hasNewFilesFormatted" ]; then
+    git add .
```

Comment:
It seems this could potentially add files that aren't intended to be added to the commit. We'd have to use `hasNewFilesFormatted` I think.

```suggestion
    git add $hasNewFilesFormatted
```